src/content/en/2021/resource-hints.md

```diff
@@ -353,7 +405,7 @@ Attempting to preload a CORS-enabled resource without including the correct `cro
   )
 }}
 
-More than half of the cases when the `crossorigin` attribute on the `rel="preload"` hint is either missing or incorrect are linked to the preloading of font files, with a total of 14,818 instances across the dataset.
+More than half of the cases when the `crossorigin` attribute on the `rel="preload"` hint is either missing or incorrect, are linked to the preloading of font files, with a total of 14,818 instances across the dataset.
```

Comment:
If you prefer being more specific (instead of "more than half"), it is 63.6% on mobile.

Reply:
I think that's good! Use it. More than half to me suggests %52 - i.e. it's just more than half, or you would have said so.

Reply:
"More than half (63.6%) of the cases..." works for me - best of both worlds!